### ISSUE TITLE
Notifications show up blank in Compiz

### ISSUE THREAD

**Issue by gabrielfin** [BelledonneCommunications/linphone-desktop#508]
Hi, I'm trying out Linphone for the first time. When I receive a call, the notification shows as a white rectangle at the right bottom edge of the screen. It seems that the notification is still clickable, because depending on where I click, it sometimes picks up or hangs up the call.

![screenshot](https://user-images.githubusercontent.com/787075/121950049-93b1eb80-cd2f-11eb-8fd7-986913878855.png)

The only thing relevant I see when runing from the command line:
```
[16:38:06:206][0x234ec90][Info]components/notifier/Notifier.cpp:192: "Create notifications:" NotificationReceivedCall_QMLTYPE_294(0x3d98a70, name="__internalWrapper", parent=0x3eaf800, geometry=0,0 0x0)
```

I'm running Ubuntu 20.04 with MATE  and Compiz. With Marco+Compton the notification works ok.

**Comment by flamelcadet** [comment]
Same problem with picom
[08:39:28:600][0x200e680][Info]"Create notifications:" NotificationReceivedCall_QMLTYPE_455(0x67f7f90, name="__internalWrapper", parent=0x56c2d50, geometry=0,0 0x0)
[08:39:33:407][0x200e680][Info]"Notify event count: 0."
[08:39:33:407][0x200e680][Info]"Notify event count: 0."
[08:39:33:601][0x200e680][Info]"Notify event count: 0."
[08:39:33:601][0x200e680][Info]"Notify event count: 0."
[08:39:37:483][0x200e680][Info]"Update (`sip:`) from chat call."
[08:39:37:487][0x200e680][Info]"Delete notification:" NotificationReceivedCall_QMLTYPE_455(0x67f7f90, name = "__internalWrapper")

**Comment by julonexus** [comment]
Should be fixed at least from 5.0.10
